### PR TITLE
Angular client to lsp-angular.el

### PR DESCRIPTION
**Pull request by elxbarbosa**
Hi,

Changes:
- add `defgroup lsp-angular`
- add angular `:install:` to `lsp-clients.json`

Issues: 
- Server not installing automatically

### CODE REVIEW

File: docs/lsp-clients.json [hunk 1]
```diff
@@ -11,9 +11,10 @@
     "name": "angular",
     "full-name": "Angular",
     "server-name": "vscode-ng-language-service",
-    "server-url": "https://github.com/angular/vscode-ng-language-service/",
+    "server-url": "https://github.com/angular/vscode-ng-language-service",
     "installation-url": "https://github.com/emacs-lsp/lsp-mode/wiki/Install-Angular-Language-server",
-    "debugger": "Not relevant"
+    "installation": "npm install -g @angular/language-service@next typescript  @angular/language-server",
+      "debugger": "Not relevant"
```
Comment (ericdallo) [inline docs/lsp-clients.json:17]:
```suggestion
    "debugger": "Not relevant"
```

**Comment by ericdallo** [comment]
Please resolve the conflict

**Comment by elxbarbosa** [comment]
would not this one be in lsp-javascript too?

**Comment by ericdallo** [comment]
IMO, no, there are a lot of js frameworks like vue.js and their heave their own file separated

File: lsp-angular.el [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+;; LSP Clients for the Angular Web application framework.
+
+;;; Code:
+
+(require 'lsp-mode)
+(require 'f)
+
+

+;;; Angular
```
Comment (ericdallo) [inline lsp-angular.el:31]:
```suggestion
```

**Pull request merged**